src/Modal/Modal.dropdownExample.js

```diff
@@ -27,16 +27,10 @@ export default class ModalDropdownDemo extends React.Component {
     return (
       <div>
         <Dropdown placement="top" width={250} trigger={<Button variant="success">Open Dropdown</Button>}>
-          <Dropdown.Header title="Dropdown" />
```

Comment:
No more need for the multi section dropdown? Do we want to use the divider here? I think we had added the dropdown to facilitate keyboard accessibility testing.

Reply:
wut